tools/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/LiferayUpgradePerspectiveFactory.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+package com.liferay.ide.ui;
+
+import org.eclipse.jdt.ui.JavaUI;
+import org.eclipse.ui.IFolderLayout;
+import org.eclipse.ui.IPageLayout;
+import org.eclipse.ui.progress.IProgressConstants;
+
+/**
+ * @author Lovett Li
+ */
+public class LiferayUpgradePerspectiveFactory extends AbstractPerspectiveFactory
+{
+    public static final String ID = "com.liferay.ide.eclipse.ui.perspective.liferaycodeupgrade";
+
+    @Override
+    public void createInitialLayout( IPageLayout layout )
+    {
+        createLayout(layout);
+        addShortcuts(layout);
+    }
+
+    private void createLayout( IPageLayout layout )
+    {
+        String editorArea = layout.getEditorArea();
+        layout.setEditorAreaVisible( false );
```

Comment:
So remove that attribute it looks like this it will show empty editor area. Is it ok? or you wanna show this empty editor area to right or somewhere. any suggestion?

<img width="1156" alt="screen shot 2016-09-21 at 23 22 50" src="https://cloud.githubusercontent.com/assets/10591143/18717615/0d7d028a-8053-11e6-91e6-11edeea7df4b.png">